### PR TITLE
halves some looping sound sounds

### PR DESCRIPTION
**Pull request by alsoandanswer**
i didn't realize it would be this easy

basically divides some numbers by 2

### CODE REVIEW

File: html/changelogs/wezzy_loopsound_lower.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: ChangeMe
```
Comment (Doxxmedearly) [inline html/changelogs/wezzy_loopsound_lower.yml]: New alias, huh?

**Comment by alsoandanswer** [comment]
!review

**Comment by NonQueueingMatt** [comment]
I have doubts about this working the way you think it will. Did you test it?

**Comment by alsoandanswer** [comment]
> 
> 
> I have doubts about this working the way you think it will. Did you test it?

Tested it, it works. Turns out this was just a case of Occam's razor.
I tried setting it to different volumes trying to very closely pay attention to changes in volume
until I realized I could just set it to 0 and see if the sound stopped

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**